src/test/shell/data/i18n/de.json

```diff
@@ -429,9 +429,7 @@
     "MEMBERS_LIST": "Selected members",
     "TABLE": {
       "USERNAME": "Username",
-      "FIRSTNAME": "First name",
```

Comment:
Next time it would be better to create a dedicated Issue / PR even for such small changes.